### ISSUE TITLE
False positive on case arrow(->) operator. 

### ISSUE THREAD

**Issue by AkshathaC17** [rrrene/credo#502]
Environment
Credo version (mix credo -v): 0.8.10
Elixir: 1.6.1
Erlang 20.2

Operating system:
MacOS Sierra version 10.12.5

What were you trying to do?
The following code:
```
   case String.length(prefixed_number) do
       10 -> prefixed_number
       _other -> full_number
    end
```
Expected outcome
No warnings

Actual outcome
"There are spaces around operators most of the time, but not here".

**Comment by rrrene** [comment]
Could you test with `0.9.0-rc3`? There's still issues to iron out, but I think we're close to release.

**Comment by AkshathaC17** [comment]
`0.9.0-rc3` resolved the issue. Thanks @rrrene.
